windows/devices/uwb/include/windows/devices/uwb/UwbConnector.hxx

```diff
@@ -226,15 +221,52 @@ private:
     std::vector<std::shared_ptr<::uwb::UwbRegisteredDeviceEventCallbacks>>
     GetResolvedDeviceEventCallbacks();
 
+    /**
+     * @brief Internal helper function to insert the callback into the hashmap of ids to callbacks
+     * You MUST have grabbed the m_eventCallbacksGate mutex prior to calling this.
+     *
+     * @param callback
+     * @return std::weak_ptr<::uwb::RegisteredCallbackToken>
+     */
+    std::shared_ptr<::uwb::RegisteredCallbackToken>
+    InsertDeviceEventCallback(std::weak_ptr<::uwb::UwbRegisteredDeviceEventCallbacks> callback);
+
+    /**
+     * @brief Internal helper function to insert the callback into the hashmap of ids to callbacks
+     * You MUST have grabbed the m_eventCallbacksGate mutex prior to calling this.
+     *
+     * @param callback
+     * @return std::weak_ptr<::uwb::RegisteredCallbackToken>

```

Comment:
The function returns `std::shared_ptr`, so the comment should be updated

Reply:
Like other function, the return type should just be changed to `std::weak_ptr<::uwb::RegisteredCallbackToken>`.